src/main/com/fulcrologic/fulcro/server/ion_middleware.clj

```diff
@@ -0,0 +1,182 @@
+(ns com.fulcrologic.fulcro.server.ion-middleware
+  "a copy of api-middleware that can be used with Datomic Cloud Ions, which use an older version of Transit"
+  (:require
+    [clojure.pprint :refer [pprint]]
+    [clojure.test :refer :all]
+    [clojure.repl :refer [doc source]]
+    [com.fulcrologic.fulcro.algorithms.transit-old :as transit]
+    [cognitect.transit :as ct]
+    [taoensso.timbre :as log])
+  (:import (java.io ByteArrayOutputStream)))
+
+(def not-found-handler
```

Comment:
Quite a few of these functions are purely duplicate code. Is this necessary?

Reply:
Yes, I hate all the duplication, too.  I’ve thought of a way to reduce the amount of duplication.  I’ll make the changes and update the PR, probably tomorrow.